shell/package.json

```diff
@@ -161,7 +161,7 @@
     "roarr": "7.0.4",
     "semver": "7.5.4",
     "@types/lodash": "4.17.5",
-    "@types/node": "~20.10.0",
+    "@types/node": "20.17.6",
```

Comment:
We can consider sticking to `20.10.8` if this is known to be working in shell. I only went with `20.17.6` because this is the latest available for node 20.